students/Ruohan/lesson03/mailroom.py

```diff
@@ -0,0 +1,95 @@
+#! /usr/bin/env python3
+
+''' simple mailroom proram for a non profit'''
+
+import sys
+
+datas = [['William Gates III',  653784.39, 2],
+         ['Mark Zuckerberg', 16396.10, 3],
+         ['Jeff Bezos', 877.33, 1],
+         ['Paul Allen', 708.42, 3]]
+
+'''make a donors' name list'''
+donors = []
+for data in datas:
+    donors.append(data[0])
+
+'''find if donor is already in the datas, if not add the name to the datas'''
```

Comment:
If you want this to be a docstring comment for this function, move this line below line 18, indented.